frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetToRemoteAction.java

```diff
@@ -156,8 +157,8 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       val dialogBuilder = MessageDialogBuilder.okCancel(
           getString("action.GitMachete.BaseResetToRemoteAction.info-dialog.title"),
           getString("action.GitMachete.BaseResetToRemoteAction.info-dialog.message.HTML").format(
-              branchName,
-              remoteTrackingBranch.getName(),
+              escapeHtml4(branchName),
```

Comment:
To be checked:
1. what happens if `null` is passed to `escapeHtml4`? NPE thrown, or null returned?
2. does Nullness Checker allow for passing a nullable value to `escapeHtml4` currently?

Reply:
<img width="460" alt="image" src="https://user-images.githubusercontent.com/61919032/197479246-fe91bb7e-5c31-4e7e-a798-6372b1566186.png">

And yes, Nullness Checker allows for passing a nullable value to `escapeHtml4`

Reply:
Already marked as `@PolyNull` in the other stub file